### DISCUSSION TITLE
The invite to discourse seems broken

### DISCUSSION THREAD

**Discussion by ovizii** [mealie-recipes/mealie#307] [General]
The link is here: https://github.com/hay-kot/mealie 

![image](https://user-images.githubusercontent.com/837902/115016950-10eaec80-9eb6-11eb-9cb9-55cc234dd72b.png)

The result:
![image](https://user-images.githubusercontent.com/837902/115017004-2a8c3400-9eb6-11eb-951a-87953b748145.png)

**Comment by hay-kot** [comment]
Bummer! I must have set the wrong limits when I created it. Here's one that should work. https://discord.gg/QuStdQGSGK
